src/styles/main.scss

```diff
@@ -0,0 +1,57 @@
+.body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.clock {
+  position: relative;
+  width: 800px;
+  height: 800px;
+  border: 5px dotted black ;
+  border-radius: 50%;
+
+  &__minute {
+    position: absolute;
+    width: 6px;
+    height: 400px;
+    left: 397px;
+    background-color: green;
+    animation: move 5s linear infinite ;
+  }
+
+  &__watch {
+    position: absolute;
+    width: 10px;
+    height: 200px;
+    left: 395px;
+    top: 200px;
+    background-color: blue;
+    animation: move 60s linear infinite;
+  }
+
+  &__center {
+    position: absolute;
+    top: 380px;
+    left: 380px;
+    width: 40px;
+    height: 40px;
+    background-color: yellow;
+    border-radius: 50%;
```

Comment:
you can create a mixin for creating a circle since you create a circle twice.

```
@mixin circle($size) {
  width: $size;
  height: $size;
  border-radius: 50%;
}
```